packages/elements/cli.mjs

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env node
-const yargs = require('yargs/yargs');
-const { hideBin } = require('yargs/helpers');
-const { errorHandler, info } = require('./scripts/helpers');
+import yargs from 'yargs/yargs';
+import { hideBin } from 'yargs/helpers';
+import { errorHandler, info } from './scripts/helpers/index.mjs';
+import { commands } from './scripts/cmd/index.mjs'
 
 yargs(hideBin(process.argv))
```

Comment:
Why are we extracting arguments but not using them at all here? SonarCloud also reports a [bug](https://sonarcloud.io/project/issues?fileUuids=AYbpkTwsQIBGOu1zkTY0&resolved=false&pullRequest=629&id=refinitiv_refinitiv-ui&open=AYbpkT7QQIBGOu1zkTee).

Reply:
It does not work if we do not have it.

Reply:
we can use `.parse()` instead.

> Calling .parse() with no arguments is equivalent to calling .argv:

https://yargs.js.org/docs/#api-reference-parseargs-context-parsecallback